### PR TITLE
Questions jg

### PR DESCRIPTION
**Pull request by JenG1**
Added images, questions.html (basic responsive grid layout), and questions.js.

### CODE REVIEW

File: config/db_config.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 require("dotenv").config()
 
 const db_obj = {
-    database: process.env.DB_NAME || "jkready",
+    database: process.env.DB_NAME || "jkready_db",
     username: process.env.DB_USER || "root",
-    password: process.env.DB_PW,
+    password: process.env.DB_PW || "Biodesi647025!",
```
Comment (kshep425) [inline config/db_config.js]: Your password shouldn't be checked in to github.

File: public/assets/index.html [hunk 2]
```diff
@@ -63,7 +63,7 @@ <h2>Instructions</h2>
 		</form>
     </div> -->
     <script src="https://code.jquery.com/jquery.js"></script>
-    <script src="./scripts/script.js" type="application/javascript"></script>
+    <script src="./scripts/animation.js" type="application/javascript"></script>
```
Comment (kshep425) [inline public/assets/index.html]: We need to make sure we update this wherever script.js is called.

File: public/assets/scripts/questions.js [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+  
+      // Trigger a user-select-change
+      container.trigger('user-select-change');
+    });
+  }
+  
+  // "Main method" which will create all the objects and render the Quiz.
+  $(document).ready(function () {
+    // Create an instance of the Quiz object
+    let quiz = new Quiz('Rabbit The Fixer');
```
Comment (kshep425) [inline public/assets/scripts/questions.js]: Change to Foxy

File: public/assets/scripts/questions.js [hunk 4]
```diff
@@ -0,0 +1,277 @@
+// Array of all the questions and choices to populate the questions. This might be saved in some JSON file or a database and we would have to read the data in.
+const all_questions = [{
```
Comment (kshep425) [inline public/assets/scripts/questions.js]: Get questions from /api/questions

File: pages/src/contact.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```
Comment (kshep425) [inline pages/src/contact.html]: The pages folder and contents can be removed.  I merged them into the handlebars already.

Comment (kshep425) [reply pages/src/contact.html]: contact is in the handlebars...needs javascript to hide until contact link is clicked.

File: pages/src/highscore.html [hunk 6]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
```
Comment (kshep425) [inline pages/src/highscore.html]: high scores is also in the handlebars...it needs to reference the api/scores though.

File: pages/src/index.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
+import Phaser from 'phaser';
```
Comment (kshep425) [inline pages/src/index.js]: This isn't needed we are no longer using phaser.

**Comment by Rmj22** [comment]
I took that out last night while I was still working I’ll push again so you
can see what I have now

**Pull request merged**